lib/multi-level.js

```diff
@@ -121,16 +146,17 @@ class MultiLevelCache {
         const value = await task;
         const cacheItem = {
           value,
-          expiry: Date.now() + this._maxAge
+          expiry: Date.now() + getValue(opts, 'maxAge', this._maxAge)
         };
 
-        if (shouldCache && typeof shouldCache !== 'function') {
+        const { shouldCache = this.shouldCache } = opts;
```

Comment:
Just to be stylistically consistent, just use `getValue` again

```js
const shouldCache = getValue(opts, 'shouldCache', this.shouldCache);
```

Reply:
sure